RELEASE.md

```diff
@@ -79,13 +96,17 @@ make release
 This process will take a couple minutes. 
 Once completed, the zipped release files will be located in the [xfra-layer0](https://console.aws.amazon.com/s3/home?region=us-west-2#&bucket=xfra-layer0&prefix=release) S3 bucket. 
 
+
 # Announce the release
+
 Once the release is available for download, send a message to
 [#xfra](https://ims-dev.slack.com/messages/xfra) with a link and a one-line
 summary of the release contents. For example:
 ```
-@here Layer0 has released v0.5.2 http://docs.xfra.ims.io/releases - which aims to be a stable version of our load balancer support.
+@here Layer0 has released v0.9.0 http://docs.xfra.ims.io/releases - which aims to be a stable version of our load balancer support.
```

Comment:
I think we are going to depreciate docs.xfra.ims.io, this should probably be the github pages url